dbuf/tests/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM {{ dbuf_image.image }}
+
+RUN useradd -m hail
+USER hail
+WORKDIR /home/hail
+RUN python3 -m pip install -U numpy
+
+ENTRYPOINT []
```

Comment:
Unnecessary?

Reply:
Again, memory vague at this point, but batch does not handle entry points correctly so it was easier to just kill the entry point inherited from batch so that I can specify the command to invoke in build.yaml.